plugins/notifications-backend/src/database/DatabaseNotificationsStore.test.ts

```diff
@@ -288,6 +299,164 @@ describe.each(databases.eachSupportedId())(
         expect(allUserNotificationsPageTwo.at(1)?.id).toEqual(id4);
         expect(allUserNotificationsPageTwo.at(2)?.id).toEqual(id3);
       });
+
+      it('should sort result', async () => {
+        const now = Date.now();
+
+        await insertNotification({
+          id: id1,
+          user,
+          origin: 'Y',
+          payload: {
+            title: 'Notification 1',
+            link: '/catalog',
+            severity: 'normal',
+            topic: 'AAA',
+          },
+          created: new Date(now - 10 * timeDelay),
+        });
+        await insertNotification({
+          id: id2,
+          user,
+          origin: 'Z',
+          payload: {
+            title: 'Notification 2',
+            link: '/catalog',
+            severity: 'normal',
+            topic: 'CCC',
+          },
+          created: new Date(now),
+        });
+        await insertNotification({
+          id: id3,
+          user,
+          origin: 'X',
+          payload: {
+            title: 'Notification 3',
+            link: '/catalog',
+            severity: 'normal',
+            topic: 'BBB',
+          },
+          created: new Date(now - 5 * timeDelay),
+        });
+
+        const notificationsCreatedAsc = await storage.getNotifications({
+          user,
+          sort: 'created',
+          sortOrder: 'asc',
+        });
+        expect(notificationsCreatedAsc.length).toBe(3);
```

Comment:
Jest can already assert length. This will improve the error message if it fails:

```suggestion
        expect(notificationsCreatedAsc). toHaveLength(3);
```